gateway/src/apicast/policy/mtls/mtls.lua

```diff
@@ -0,0 +1,41 @@
+local policy = require('apicast.policy')
+local _M = policy.new('mtls')
+local X509 = require('resty.openssl.x509')
+local errors = require('apicast.errors')
+local re_split = require('ngx.re').split
+
+local mt = {
+  __index = _M
+}
+
+local function encode(bytes)
+  local encoded = ngx.encode_base64(bytes)
+  return re_split(encoded, '=')[1]
+end
+
+local function check_certificate(cert, cnf)
+  if not (cert and cnf) then return false end
+
+  if encode(cert:digest('SHA256')) == cnf['x5t#S256'] then return true end
+
+  return false
+end
+
+function _M.new()
+  return setmetatable({
+  }, mt)
+end
+
+function _M:access(context)

```

Comment:
To use `context.jwt`, we need to specify `:`.

Reply:
you can use function `_M.access(_, context)` no?

Reply:
Changed.